e2e/cypress/integration/system_console/ui_and_api/notifications_spec.js

```diff
@@ -0,0 +1,79 @@
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('System Console', () => {
+    before(() => {
+        // # Update to default config
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                PushNotificationContents: 'full',
+            },
+        });
+
+        // # Login as sysadmin and visit Notifications admin console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/environment/notifications');
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Notifications');
+    });
+
+    it('Push Notification Contents', () => {
+        // * Verify that setting is visible and matches text content
+        cy.findByTestId('EmailSettings.PushNotificationContents').
+            scrollIntoView().should('be.visible').
+            find('label').should('be.visible').and('have.text', 'Push Notification Contents:');
+
+        // * Verify that the help text is visible and matches text content
+        cy.findByTestId('EmailSettings.PushNotificationContentshelp-text').should('be.visible').within((el) => {
+            const contents = [
+                'Generic description with only sender name',
+                ' - Includes only the name of the person who sent the message in push notifications, with no information about channel name or message contents. ',
+                'Generic description with sender and channel names',
+                ' - Includes the name of the person who sent the message and the channel it was sent in, but not the message contents. ',
+                'Full message content sent in the notification payload',
+                ' - Includes the message contents in the push notification payload that is relayed through Apple\'s Push Notification Service (APNS) or Google\'s Firebase Cloud Messaging (FCM). It is ',
+                'highly recommended',
+                ' this option only be used with an "https" protocol to encrypt the connection and protect confidential information sent in messages.',
+                'Full message content fetched from the server on receipt',
+                ' - The notification payload relayed through APNS or FCM contains no message content, instead it contains a unique message ID used to fetch message content from the server when a push notification is received by a device. If the server cannot be reached, a generic notification will be displayed.',
+            ];
+            cy.wrap(el).should('have.text', contents.join(''));
+
+            cy.wrap(el).find('strong').eq(0).should('have.text', contents[0]);
+            cy.wrap(el).find('strong').eq(1).should('have.text', contents[2]);
+            cy.wrap(el).find('strong').eq(2).should('have.text', contents[4]);
+            cy.wrap(el).find('strong').eq(3).should('have.text', contents[6]);
+            cy.wrap(el).find('strong').eq(4).should('have.text', contents[8]);
```

Comment:
As advised by Bob, we do not need to include cy.wrap again, inside the within block. This code could be simplified like this:

```
            cy.get('strong').eq(0).should('have.text', contents[0]);
            cy.get('strong').eq(1).should('have.text', contents[2]);
            cy.get('strong').eq(2).should('have.text', contents[4]);
            cy.get('strong').eq(3).should('have.text', contents[6]);
            cy.get('strong').eq(4).should('have.text', contents[8]);
```

Reply:
Done, PTAL